docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+Geotrellis Quickstart Guide
+===========================
+
+Geotrellis is a large and evolving package, which makes it difficult to maintain current documentation.  To make up for this deficiency, this quickstart guide will provide a set of documents describing case studies that accomplish some simple goals.  These examples are meant to highlight the fundamental Geotrellis classes as well as show some basic Scala techniques for those new to the language.
+
+Getting Started
+---------------
+
+In the following, we'll be providing some Scala code samples that make use of the GeoTrellis library.  This requires having a running Scala instance with GeoTrellis set as a library dependency.  It is recommended that the user download and install SBT, the Scala Build Tool.  Once that is installed, it is recommended that you use `git` to clone the repository at `github.com/geotrellis/geotrellis-sbt-template repository`.  Once available, from the root directory of the cloned repo, MacOSX and Linux users may launch the sbt script contained therein.  This will start an SBT session.  Once SBT is started, it is advisable to issue the `console` command; this will start the Scala interpreter.  At this point, you should be able to issue the command `import geotrellis.vector._` without raising an error.
+
+Basic Geotrellis Types
+----------------------
+
+Geotrellis itself is a library for working with raster data over maps.  Raster data is essentially a regular, rectangular grid containing scalar values (i.e., numerical data) in each grid cell.
+
+The base type for these grid structures is `CellGrid` which offers basic dimension information.  The `Tile` trait gives the general interface for working with data over a grid.  Actual usage of Tiles requires an understanding of the NoData handling options defined in `CellType` and described in more detail [elsewhere](../raster/celltype.md) in the documentation.  Generally speaking, though, for Tiles containing integer data types (including bits and bytes), the `IntArrayTile` companion object is used to construct Tiles, while floating point Tiles are created using the `DoubleArrayTile` companion object.
+
+A Tile needs additional context if it is to represent data on a map; specifically, it needs an `Extent` which defines the map coordinates of the corners of the Tile.  `Extent(xmin,ymin,xmax,ymax)` represents the bounding box of the Tile *expressed in some coordinate system*.  To apply the Tile to a map, it'll be necessary to define which coordinate system was used, though that will happen at a later stage.  A Tile and Extent come together to form a `Raster.`
```

Comment:
`To apply the Tile to a map` is a bit confusing, perhaps "In order for the Tile to represent data on a portion of some map`?